Pear/View/EndUserLicenseAgreementView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  EndUserLicenseAgreementView.swift
+//  Pear
+//
+//  Created by Vian Nguyen on 9/18/22.
+//  Copyright © 2022 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+protocol UserAgreementDelegate: AnyObject {
+    func presentUserLicenseAgreementWebView(url: URL)
+}
+
+class EndUserLicenseAgreementView: UIView {
+        
+    // MARK: - Private View Vars
+    private let acceptButton = DynamicButton()
+    private let descriptionLabel = UILabel()
+    private let headerLabel = UILabel()
+    private let licenseButton = UIButton()
+    private let pearImageView = UIImageView()
+
+    // MARK: - Private Data Vars
+    private var delegate: UserAgreementDelegate?
+    private let licenseURL = "https://www.cornellappdev.com/license/pear"
+    private let licenseButtonText = NSMutableAttributedString.init(string: "End User License Agreement")
+    
+    init(delegate: UserAgreementDelegate?) {
+        super.init(frame: .zero)
+        self.delegate = delegate
+        setupViews();
+        setupConstraints();
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupViews() {
+        backgroundColor = .backgroundLightGreen
```

Comment:
Just curious, but does Pear have a thematic color palette. For example Eatery and Volume has a static constant files for the colors. I noticed that you seem to be using system colors throughout the file. Perhaps it might be more extendable if you were to factor this out?

Reply:
Yes, we have a UIColor extension, so I believe backgroundLightGreen and darkGreen are defined or overridden with custom colors.